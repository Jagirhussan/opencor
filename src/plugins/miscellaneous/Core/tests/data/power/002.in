<math xmlns="http://www.w3.org/1998/Math/MathML">
    <apply>
        <eq/>
        <ci>a</ci>
        <apply>
            <power/>
            <apply>
                <minus/>
                <ci>b</ci>
            </apply>
            <cn>3</cn>
        </apply>
    </apply>
</math>
