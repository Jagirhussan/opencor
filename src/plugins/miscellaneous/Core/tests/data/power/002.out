<math xmlns="http://www.w3.org/1998/Math/MathML">
    <mrow>
        <mi mathvariant="italic">a</mi>
        <mo>=</mo>
        <msup>
            <mrow>
                <mo>(</mo>
                <mo>âˆ’</mo>
                <mi mathvariant="italic">b</mi>
                <mo>)</mo>
            </mrow>
            <mn>3</mn>
        </msup>
    </mrow>
</math>
